.github/workflows/build.yml

```diff
@@ -22,6 +22,14 @@ jobs:
       - name: Test
         run: go test -v ./...
 
+      - name: Check Format
+        run: |
+          bad_files="$(gofmt -s -l .)"
+          if [[ -n $bad_files ]]; then
+              printf -- "Some files are not formatted: %s\n" "$bad_files"
```

Comment:
should output it as an error: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-error-message